p2p/discovery/mdns.go

```diff
@@ -152,8 +165,10 @@ func (m *mdnsService) pollForEntries(ctx context.Context) {
 }
 
 func (m *mdnsService) handleEntry(e *mdns.ServiceEntry) {
-	log.Debugf("Handling MDNS entry: %s:%d %s", e.AddrV4, e.Port, e.Info)
-	mpeer, err := peer.IDB58Decode(e.Info)
+	// pull out the txt
+	info := strings.Join(e.Text, "|")
```

Comment:
is this designed to fail if there are less than or more than one entry?

Reply:
to be clear, because of the decoding afterwards

Reply:
Hm. Yeah, that looks wrong. We should just take the first entry and ignore the rest.